src/Pos/Block/Logic/Internal.hs

```diff
@@ -111,6 +117,9 @@ applyBlocksUnsafeDo
 applyBlocksUnsafeDo blunds pModifier = do
     -- Note: it's important to put blocks first
     mapM_ putToDB blunds
+    -- If program is interrupted now, we will rollback all wallet sets
```

Comment:
Also I am not sure it's a correct place for such comments, because this code doesn't know anything about wallet sets. But it's minor.